webpack/webpack.common.js

```diff
@@ -1,83 +1,101 @@
 const webpack = require('webpack');
+const { BaseHrefWebpackPlugin } = require('base-href-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const rxPaths = require('rxjs/_esm5/path-mapping');
+const AngularCompilerPlugin = require('@ngtools/webpack').AngularCompilerPlugin;
 
 const utils = require('./utils.js');
 
-module.exports = (options) => ({
-    resolve: {
-        extensions: ['.ts', '.js'],
-        modules: ['node_modules'],
-        alias: {
-            app: utils.root('src/main/webapp/app/'),
-            ...rxPaths()
+module.exports = options => ({
+  resolve: {
+    extensions: ['.ts', '.js'],
+    modules: ['node_modules'],
+    mainFields: ['es2015', 'browser', 'module', 'main'],
```

Comment:
I suggest removing `es2015` for now as per, https://github.com/jhipster/generator-jhipster/pull/11580 since otherwise users who use Firefox will have problems. We can put it back after sometime Firefox 77 is released.

Reply:
good catch, I haven't noticed this on my firefox browser. It could be due to that fact that I am using old `v72.x`

Reply:
should be good now